lib/lightning_web/router.ex

```diff
@@ -82,10 +82,13 @@ defmodule LightningWeb.Router do
 
     resources "/projects", API.ProjectController, only: [:index, :show] do
       resources "/jobs", API.JobController, only: [:index, :show]
-      resources "/workflows", API.WorkflowsController, except: [:delete]
       # resources "/runs", API.RunController, only: [:index, :show]
     end
 
+    resources "/v1/projects", API.ProjectController, only: [:index, :show] do
```

Comment:
:exclamation: I am worried about this because Stu was keen on implementing header-based versioning for the Collections API

I strongly  recommend we either implement this as a header, or revert the change and wait discuss with Stu next week.

Reply:
We should for him, thanks!

Reply:
One factor to take into consideration is that the Collections is an internal API while the workflows will be available for other kinds of integrations. But if we are not sure how stable the API will be on the fields and structure the Headers versioning would be a better alternative (otherwise, considering the evolution of Triggers, Edges and Jobs fields the URL is the most explicit and clear one).